? testtools/source/bridgetest/cli/cli_bridgetest_inprocess.cs.origish
? testtools/source/bridgetest/cli/mcs-failure
? testtools/source/bridgetest/cli/mcs-failure.tar.gz
Index: testtools/source/bridgetest/cli/cli_bridgetest_inprocess.cs
===================================================================
RCS file: /cvs/udk/testtools/source/bridgetest/cli/cli_bridgetest_inprocess.cs,v
retrieving revision 1.5
diff -u -r1.5 cli_bridgetest_inprocess.cs
--- testtools/source/bridgetest/cli/cli_bridgetest_inprocess.cs	20 Sep 2004 14:27:51 -0000	1.5
+++ testtools/source/bridgetest/cli/cli_bridgetest_inprocess.cs	11 May 2005 14:19:54 -0000
@@ -139,26 +139,8 @@
                     new uno.Any(
                         typeof (XSingleComponentFactory),
                         new Factory(
-                            typeof (vb_testobj.VBBridgeTestObject),
-                            "com.sun.star.test.bridge.cli_uno.VbTestObject" ) ) );
-                xSet.insert(
-                    new uno.Any(
-                        typeof (XSingleComponentFactory),
-                        new Factory(
-                            typeof (cpp_bridgetest.BridgeTest),
-                            "com.sun.star.test.bridge.cli_uno.CppBridgeTest" ) ) );
-                xSet.insert(
-                    new uno.Any(
-                        typeof (XSingleComponentFactory),
-                        new Factory(
                             typeof (cs_testobj.BridgeTest),
                             "com.sun.star.test.bridge.cli_uno.CsBridgeTest" ) ) );
-                xSet.insert(
-                    new uno.Any(
-                        typeof (XSingleComponentFactory),
-                        new Factory(
-                            typeof (vb_bridetest.BridgeTest),
-                            "com.sun.star.test.bridge.cli_uno.VbBridgeTest" ) ) );
             
                 // I.
                 // direct unbridged test
Index: testtools/source/bridgetest/cli/makefile.mk
===================================================================
RCS file: /cvs/udk/testtools/source/bridgetest/cli/makefile.mk,v
retrieving revision 1.11
diff -u -r1.11 makefile.mk
--- testtools/source/bridgetest/cli/makefile.mk	16 Dec 2004 12:49:01 -0000	1.11
+++ testtools/source/bridgetest/cli/makefile.mk	11 May 2005 14:19:54 -0000
@@ -74,10 +74,9 @@
 .INCLUDE :  settings.mk
 DLLPRE =
 
-.IF "$(COM)$(COMEX)" == "MSC8" || "$(COM)$(COMEX)"=="MSC10"
 
 # ------------------------------------------------------------------
-.IF "$(GUI)"=="WNT"
+.IF "$(GUI)"=="WINNT"
 MY_DLLPOSTFIX=.dll
 DESTDIR=$(BIN)
 BATCH_SUFFIX=.bat
@@ -106,20 +105,20 @@
 
 #-----------------------
 
-CFLAGS += -clr -AI $(OUT)$/bin -AI $(SOLARBINDIR)
+# CFLAGS += -clr -AI $(OUT)$/bin -AI $(SOLARBINDIR)
 
-SLOFILES= \
-	$(SLO)$/cli_cpp_bridgetest.obj		
+#SLOFILES= \
+#	$(SLO)$/cli_cpp_bridgetest.obj		
 
-SHL1OBJS = $(SLOFILES)
+#SHL1OBJS = $(SLOFILES)
 
-SHL1TARGET = $(TARGET)
+#SHL1TARGET = $(TARGET)
 
-SHL1STDLIBS = \
-	mscoree.lib
+#SHL1STDLIBS = \
+#	mscoree.lib
 
-SHL1DEF = $(MISC)$/$(SHL1TARGET).def
-DEF1NAME = $(SHL1TARGET)
+#SHL1DEF = $(MISC)$/$(SHL1TARGET).def
+#DEF1NAME = $(SHL1TARGET)
 
 
 # --- Targets ------------------------------------------------------
@@ -136,7 +135,7 @@
 CLI_CPPUHELPER = $(SOLARBINDIR)$/cli_cppuhelper.dll
 CLI_TYPES_BRIDGETEST = $(BIN)$/cli_types_bridgetest.dll
 
-CSCFLAGS = -warnaserror+ -incremental-
+CSCFLAGS = -incremental-
 VBC_FLAGS = -warnaserror+
 .IF "$(debug)" != ""
 CSCFLAGS += -debug+ -checked+ -define:DEBUG -define:TRACE
@@ -158,7 +157,7 @@
 		-reference:$(CLI_TYPES_BRIDGETEST) \
 		-reference:$(CLI_URE) \
 		-reference:$(CLI_BASETYPES) \
- 		-reference:$(CLI_TYPES) \
+		-reference:$(CLI_TYPES) \
 		cli_cs_testobj.cs cli_cs_multi.cs
 
 $(DESTDIR)$/cli_cs_bridgetest.uno.dll : \
@@ -214,9 +213,6 @@
 		cli_bridgetest_inprocess.cs \
 		$(DESTDIR)$/cli_cs_bridgetest.uno.dll \
 		$(DESTDIR)$/cli_cs_testobj.uno.dll \
-		$(DESTDIR)$/cli_vb_bridgetest.uno.dll \
-		$(DESTDIR)$/cli_vb_testobj.uno.dll \
-		$(DESTDIR)$/cli_cpp_bridgetest.uno.dll \
 		$(CLI_BASETYPES) \
 		$(CLI_TYPES) \
 		$(CLI_URE) \
@@ -229,9 +225,6 @@
 		-reference:$(CLI_CPPUHELPER) \
 		-reference:$(DESTDIR)$/cli_cs_bridgetest.uno.dll \
 		-reference:$(DESTDIR)$/cli_cs_testobj.uno.dll \
-		-reference:$(DESTDIR)$/cli_vb_bridgetest.uno.dll \
-		-reference:$(DESTDIR)$/cli_vb_testobj.uno.dll \
-		-reference:$(DESTDIR)$/cli_cpp_bridgetest.uno.dll \
 		cli_bridgetest_inprocess.cs
 	$(GNUCOPY) -p cli_bridgetest_inprocess.ini $(DESTDIR)
 	$(GNUCOPY) -p $(CLI_CPPUHELPER) $(DESTDIR)$/$(CLI_CPPUHELPER:f)
@@ -239,5 +232,4 @@
 	$(GNUCOPY) -p $(CLI_TYPES) $(DESTDIR)$/$(CLI_TYPES:f)
 	$(GNUCOPY) -p $(CLI_URE) $(DESTDIR)$/$(CLI_URE:f)
 
-.ENDIF
 
Index: testtools/source/bridgetest/idl/makefile.mk
===================================================================
RCS file: /cvs/udk/testtools/source/bridgetest/idl/makefile.mk,v
retrieving revision 1.4
diff -u -r1.4 makefile.mk
--- testtools/source/bridgetest/idl/makefile.mk	20 Aug 2004 09:16:39 -0000	1.4
+++ testtools/source/bridgetest/idl/makefile.mk	11 May 2005 14:19:54 -0000
@@ -81,7 +81,6 @@
     + $(RM) $@
     $(REGMERGE) $@ /UCR $(MISC)$/$(TARGET)$/bridgetest.urd
 
-.IF "$(GUI)" == "WNT"
 
 CLIMAKERFLAGS =
 .IF "$(debug)" != ""
@@ -97,4 +96,3 @@
 		-X $(SOLARBINDIR)$/types.rdb $< 
     $(TOUCH) $@
 
-.ENDIF # GUI, WNT
